ce/api/multistats.py

```diff
@@ -95,6 +95,13 @@ def multistats(
             }
     """
 
+    # we don't want people requesting statistics on percentile datasets;
+    # that would be meaningless.
+    if climatological_statistic == "percentile":
+        raise Exceptions(
+            "Statistical calculations are not meaningful on percentile datasets."
+        )
+
```

Comment:
If I understand correctly, `multistats` computes statistics as requested by the param `climatological_statistic` (which can take values of `"mean"`, `"standard_deviation"`, `"percentile"`) on the base dataset. And we are concerned to exclude the case of (any) statistic over a base dataset that is already a percentile? If so, I don't understand how this condition checks that.

Reply:
`climatological_statistic` is a parameter describing a dataset; it describes the aggregation that was done to make a climatology. So this parameter, along with things like `model` and `variable`, indicates which dataset(s) the caller wants statistics from.

`multistat` always returns the same set of statistics (mean, stdev, min, max, median) on whatever datasets it is working on.

This check raises an error if the user attempts to calculate those statistics on a dataset which was created as an ensemble percentile, on grounds that these statistics are meaningless and the caller doesn't want them.

Reply:
Ah, I understand now. Sorry for the distraction. LGTM!